src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Revisions.cshtml

```diff
@@ -46,7 +46,7 @@
             @foreach (var revision in Model.Review.Revisions)
             {
                 <li class="list-group-item">
-                    @revision.Name <span class="font-italic" date="@revision.CreationDate"></span>
+                    @revision.Name <span class="font-italic font-weight-lighter revision-time-author" revision="@revision"></span>
```

Comment:
Use a separate span for author + use built-in spacing classes https://getbootstrap.com/docs/4.4/utilities/spacing/

Reply:
Fixed